src/app/perspective-status.component.ts

```diff
@@ -655,37 +655,20 @@ export class PerspectiveStatus implements OnChanges, AfterViewInit, AfterViewChe
     return updateShapeAnimationTimeline;
   }
 
-  toggleScore(): void {
-    if (this.isPlayingShowOrHideDetailsAnimation) {
-      return;
-    }
-    if (this.showScore) {
-      this.getHideDetailsAnimation().play();
-    } else {
-      this.getShowDetailsAnimation().play();
-    }
-  }
-
   setShowMoreInfo(showMoreInfo: boolean): void {
     this.getTransitionToLayerAnimation(
       showMoreInfo ? 1 : 0, LAYER_TRANSITION_TIME_SECONDS).play();
   }
 
+  // TODO(rachelrosen): Add accessibility labeling for the emoji state.
   getAnimationA11yLabel(showScore: boolean,
-                        isPlayingLoadingAnimation: boolean,
-                        isPlayingShowOrHideDetailsAnimation: boolean): string {
+                        isPlayingLoadingAnimation: boolean): string {
     if (isPlayingLoadingAnimation) {
       return "Computing score animation";
-    } else if (isPlayingShowOrHideDetailsAnimation && showScore) {
-        return "Collapsing score view animation";
-    } else if (isPlayingShowOrHideDetailsAnimation && !showScore) {
-        return "Expanding score view animation";
     } else if (showScore) {
-      return (this.getAccessibilityDescriptionForShape(this.currentShape) +
-              "Select to collapse score view");
+      return this.getAccessibilityDescriptionForShape(this.currentShape);
```

Comment:
Why bother with this else if statement as its the same as else case? 

Maybe also push the "Computing score animation" value into the `getAccessibilityDescriptionForShape` so all the functionality is in one place?

Reply:
Oh, right. Fixed.

I actually don't want to push it into getAccessibilityDescriptionForShape because I want to have this function to later delegate between descriptions for emoji/shape mode (the TODO). On second thought, I'll just do that now.

Reply:
ok!